### PR TITLE
Feature: New todo item created and redirects to detail page

### PR DESCRIPTION
**Pull request by agonzalez0515**

### CODE REVIEW

File: src/main/java/webserver/Request.java [hunk 1]
```diff
@@ -3,13 +3,28 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 
+import java.util.HashMap;
+
 public class Request {
+    private static final String EMPTY_SPACE = " ";
```
Comment (tbaik) [inline src/main/java/webserver/Request.java]: This class seems to have more than one responsibility. What do you think about splitting this up??

Comment (agonzalez0515) [reply src/main/java/webserver/Request.java]: Should I move the parsing to another class?

Comment (tbaik) [reply src/main/java/webserver/Request.java]: Yes, I think that'd be good!

Comment (agonzalez0515) [reply src/main/java/webserver/Request.java]: Just pushed my changes. I moved everything request related to a request package and moved the parsing to it's own class. The Request class only has a builder and getters now.

**Pull request closed without merging**